src/outputs/cast.c

```diff
@@ -439,7 +439,7 @@ static struct cast_session *cast_sessions;
 static struct cast_master_session *cast_master_session;
 //static struct timeval heartbeat_timeout = { HEARTBEAT_TIMEOUT, 0 };
 static struct timeval reply_timeout = { REPLY_TIMEOUT, 0 };
-static struct media_quality cast_quality_default = { CAST_QUALITY_SAMPLE_RATE_DEFAULT, CAST_QUALITY_BITS_PER_SAMPLE_DEFAULT, CAST_QUALITY_CHANNELS_DEFAULT };
+static struct media_quality cast_quality_default = { CAST_QUALITY_SAMPLE_RATE_DEFAULT, CAST_QUALITY_BITS_PER_SAMPLE_DEFAULT, CAST_QUALITY_CHANNELS_DEFAULT, 0 };
```

Comment:
Have you tested that this works? As I understand this means that the Opus encoder will be called with 0 as bit_rate, instead of the current default. How does the encoder deal with that?

Reply:
Yes this works - it was tested with a first gen Chromecast (not Chromecast Audio) plugged into the TV that identifies itself like:
```
cast: Device 'cast#00' does not support app id '85CDB22F', trying '0F5096E8' instead
```
I don't know how to determine the output quality .. when we do this with the MP3 stream it appears that ffmpeg goes with 128kbps which is/was the default from the `AVCodecContext`